.changeset/good-beds-occur.md

```diff
@@ -0,0 +1,6 @@
+---
+'@eth-optimism/core-utils': patch
```

Comment:
Actually might be worth making `core-utils` a minor patch because we're removing stuff from the API

Reply:
I agree with this, its unlikely that anybody besides us were using those functions but still best practice